src/rdo.rs

```diff
@@ -43,6 +44,41 @@ pub struct RDOPartitionOutput {
 }
 
 // Sum of Squared Error for a wxh block
+fn cdef_dist_wxh_8x8(src1: &PlaneSlice, src2: &PlaneSlice) -> u64 {
+  let coeff_shift = 0;
+  let mut sum_s: i32 = 0;
+  let mut sum_d: i32 = 0;
+  let mut sum_s2: i64 = 0;
+  let mut sum_d2: i64 = 0;
+  let mut sum_sd: i64 = 0;
+  for j in 0..8 {
+    for i in 0..8 {
+      let s = src1.p(i, j) as i32;
+      let d = src2.p(i, j) as i32;
+      sum_s += s;
+      sum_d += d;
+      sum_s2 += (s*s) as i64;
+      sum_d2 += (d*d) as i64;
+      sum_sd += (s*d) as i64;
+    }
+  }
+  let svar = (sum_s2 - ((sum_s as i64 * sum_s as i64 + 32) >> 6)) as f64;
+  let dvar = (sum_d2 - ((sum_d as i64 * sum_d as i64 + 32) >> 6)) as f64;
+  let sse = (sum_d2 + sum_s2 - 2 * sum_sd) as f64;
+  let ssim_boost = 0.5_f64 * (svar + dvar + (400 << 2 * coeff_shift) as f64) / f64::sqrt((20000 << 4 * coeff_shift) as f64 + svar * dvar);
```

Comment:
Can we get a comment or named constants to explain where the 400 and 20000 come from?